contracts/tgrade-validator-voting/src/contract.rs

```diff
@@ -218,6 +232,14 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
         Voter { address } => to_binary(&query_voter(deps, address)?),
         ListVoters { start_after, limit } => to_binary(&list_voters(deps, start_after, limit)?),
         GroupContract {} => to_binary(&query_group_contract(deps)?),
+        ListTextProposals { start_after, limit } => {
+            to_binary(&list_text_proposals::<ValidatorProposal>(
+                deps,
+                env,
+                start_after,
+                align_limit(limit),
```

Comment:
align_limit. I didn't see that before. Nice helper.

Reply:
It was there already, I only used it :+1: